demos/buildah_multi_stage.sh

```diff
@@ -60,7 +68,11 @@ buildah rm -a
 read -p "${green}Let's build the container with Buildah, first GoLang${reset}"
 buildcntr=$(buildah from golang:1.7.3)
 read -p "${green}Let's mount the container getting the root directory${reset}"
-buildmnt=$(buildah mount $buildcntr) 
+if is_rootless; then
+ buildmnt=$(buildah unshare buildah mount $buildcntr)
```

Comment:
I am not sure this will work, since when buildah unshare completes the mount point will be removed.  The entire script needs to be run within buildah unshare.

Reply:
Moved to running entire script in unshared session.